app/src/main/java/com/dkanada/gramophone/adapter/song/SongAdapter.java

```diff
@@ -159,6 +170,10 @@ protected String getSongText(Song song) {
         return MusicUtil.getSongInfoString(song);
     }
 
+    protected boolean getSongCached(Song song) {
```

Comment:
Since this is only used in one place right now I'd rather just call the Dao without a method.

Reply:
Will do.